xilem/Cargo.toml

```diff
@@ -58,7 +68,9 @@ accesskit.workspace = true
 tokio = { version = "1.39.1", features = ["rt", "rt-multi-thread", "time"] }
 
 [dev-dependencies]
-accesskit_winit.workspace = true
+# accesskit_winit.workspace = true
```

Comment:
Not necessary because this is already in masonry? (Could be removed in that case I guess)

Reply:
It's not clear to me why this dependency was required. I presume it was part of the #325/ #417 stuff (seriously, GitHub has deleted that?)

Reply:
What happened there? Bug in Github, outage? I don't think they usually delete PRs?

Reply:
The PR author's account got deleted.

Reply:
Do you know what the PR was about?

Reply:
(answering my own question, it's the "cfagot" stuff, argh they should just recover his account...)

It was "Allow external event loop to drive masonry and xilem (#417)"